data/json/items/comestibles.json

```diff
@@ -2528,9 +2550,18 @@
     "id": "mannwurst",
     "copy-from": "sausage",
     "name": "Mannwurst",
+    "flags": [ "EATEN_HOT", "SMOKED" ],
     "description": "A hefty long pork sausage that has been cured for long term storage.  Very tasty, if you're in the market for human flesh.",
```

Comment:
_"cured"_ could be replaced with _"cured and smoked"_ to be consistent with sausage.